src/powerquery-formatter/serializeV2.ts

```diff
@@ -0,0 +1,566 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as PQP from "@microsoft/powerquery-parser";
+import { Ast } from "@microsoft/powerquery-parser/lib/powerquery-parser/language";
+
+import { SerializeParameterV2 } from "./themes";
+
+import { CommentCollection, CommentCollectionMap, LinearLengthMap, SerializeParameterMapV2 } from "./passes";
+import { IndentationLiteral, NewlineLiteral, TriedSerialize } from "./serialize";
+import { getLinearLengthV2 } from "./passes/utils/linearLengthV2";
+import { NodeKind } from "@microsoft/powerquery-parser/lib/powerquery-parser/language/ast/ast";
+
+// const GLOBAL_HEADING_WHITE_SPACE_REG: RegExp = /^[ \t]+/g;
```

Comment:
Can this be removed?

Reply:
definitely, i missed the big screen in the office